Support new Flank options without updating Fladle (#214)
runningcode/fladle: docs/configuration.md

It will then expand each given scenario label into a list of scenario numbers marked with that label, and execute those scenarios.

=== "Groovy"

>     ```groovy

Reviewer: thanks for fixing this!

Author: no problem 👍

fail-fast matrices and support is more limited because of that expectation.

=== "Groovy"
    ```groovy
    failFast = true
    ```

=== "Kotlin"
    ```kotlin
    failFast.set(true)
    ```

### additionalFlankOptions
Appending additional option to flank root yaml. This option is useful when you would like to test options before official fladle support is added.

> before it is available on Fladle. Supports both single and multiple properties.

Reviewer: ```suggestion
Multiple options are supported.
```

Single option

=== "Groovy"
    ```groovy
    additionalFlankOptions = "new-property: true"
    ```

=== "Kotlin"
    ```kotlin
    additionalFlankOptions.set("new-property: true")
    ```

Multiple options

=== "Groovy"
    ```groovy

>     additionalFlankOptions = "new-property: true\nother-new-property: force"

Reviewer: ```suggestion
    additionalFlankOptions = """
      new-property: true
      other-new-property: force
    """.stripIndent()
```

    ```

=== "Kotlin"
    ```kotlin
    additionalFlankOptions.set("""
        new-property: true
        other-new-property: force
    """.trimIndent())
    ```

### additionalGcloudOptions
Allow appending additional config to gcloud root yaml. This option is useful when you would like to test option
before it is available on Fladle. Supports both single and multiple properties.

Single option

=== "Groovy"
    ```groovy
    additionalGcloudOptions = "new-property: true"
    ```

=== "Kotlin"
    ```kotlin
    additionalGcloudOptions.set("new-property: true")
    ```

Multiple options

=== "Groovy"
    ```groovy
    additionalGcloudOptions = """
        new-property: true

>         other-new-property: force""".stripIndent()

Reviewer: ```suggestion
        other-new-property: force
      """.stripIndent()
```

    ```
    failFast.set(true)
    ```

> ### additionalFlankConfig

Reviewer: Can we name this and all the other configuration options "options" instead of "config"?
```suggestion
### additionalFlankOptions
```

Author: sure, renamed

> Allow appending additional config to flank root yaml. This option is useful when you would like to test option

Reviewer: ```suggestion
Appending additional option to flank root yaml. This option is useful when you would like to test options before official fladle support is added.
```

before it is available on Fladle. Supports both single and multiple properties.

Single property

=== "Groovy"

> ```
> additionalFlankConfig = "new-property: true"
> ```

Reviewer: Please add the spacing to all of the documentation so that the mkdocs formatting works.
```suggestion
    ```
    additionalFlankConfig = "new-property: true"
    ```
```

Author: fixed

=== "Kotlin"
```
additionalFlankConfig.set("new-property: true")
```

> Multiple properties

Reviewer: Thanks for adding the single and multiple property examples. Can you make sure this looks right? You can test the documentation with `mkdocs serve`.
```suggestion
Multiple options
```

Author: works good

=== "Groovy"
```
additionalFlankConfig = "new-property: true\nother-new-property: force"
```

=== "Kotlin"
```
additionalFlankConfig.set("""
    new-property: true
    other-new-property: force
""".trimIndent())
```

### additionalGcloudConfig
Allow appending additional config to gcloud root yaml. This option is useful when you would like to test option
before it is available on Fladle. Supports both single and multiple properties.

Single property

=== "Groovy"
```
additionalGcloudConfig = "new-property: true"
```

=== "Kotlin"
```
additionalGcloudConfig.set("new-property: true")
```

Multiple properties

=== "Groovy"
```

> additionalGcloudConfig = "new-property: true\nother-new-property: force"

Reviewer: ```suggestion
    additionalGcloudConfig = """
        new-property: true
        other-new-property: force""".stripIndent()
```

Author: fixed
